vyos/vyos-1x
# containers: T2216: Add podman for containers

## Change Summary

Ability to create and remove containers and their networks.  Use "podman" for it.

## Types of changes


## Related Task(s)

* https://phabricator.vyos.net/T2216

## Related commits
https://github.com/vyos/vyos-build/pull/156
https://github.com/vyos/vyatta-cfg/pull/35

## Component(s) name

containers
## Proposed changes


## How to test
```
set container name alp01 allow-host-networks
set container name alp01 image 'alpine'
set container name alp02 image 'alpine'
set container name alp02 network NET01
set container network NET01 ipv4-prefix '10.0.0.0/24'
commit
```
Expect 2 containers
```
vyos@r4-roll:~$ show container 
CONTAINER ID  IMAGE                            COMMAND  CREATED         STATUS             PORTS   NAMES
010c511e3ebd  docker.io/library/alpine:latest  /bin/sh  12 seconds ago  Up 12 seconds ago          alp01
cb1751dca171  docker.io/library/alpine:latest  /bin/sh  11 seconds ago  Up 12 seconds ago          alp02


```

## Checklist:
